Jenkinsfile

```diff
@@ -39,5 +39,21 @@ parallel (
             manifests.buildManifest("rpi-qtauto", "core-image-pelux-qtauto-neptune")
         }
     }
```

Comment:
You are missing a comma here.
Is this rebased to the latest? I think there should also be a qemu image here?